samwise.js

```diff
@@ -1,11 +1,12 @@
 /**
```

Comment:
SCRIPT-Consider renaming this variable to something more descriptive for better readability.